modules/claims_api/spec/controllers/concerns/claims_api/v2/revised_disability_compensation_validation_spec.rb

```diff
@@ -342,5 +342,90 @@ def initialize(form_attributes)
         end
       end
     end
+
+    # FES Val Section 5.b: mailingAddress validations - PR 2 Country Validation
+    context 'mailingAddress country validation' do
+      context 'when country is invalid' do
+        let(:form_attributes) do
+          base_form_attributes.merge(
+            'veteranIdentification' => {
+              'mailingAddress' => {
+                'addressLine1' => '123 Main St',
+                'city' => 'London',
+                'country' => 'INVALID_COUNTRY',
+                'internationalPostalCode' => 'SW1A 1AA'
+              }
+            }
+          )
+        end
+
+        before do
+          allow_any_instance_of(described_class).to receive(:valid_countries).and_return(%w[USA GBR CAN])
+        end
+
+        it 'returns validation error' do
+          errors = subject.validate_form_526_fes_values
+          expect(errors).to be_an(Array)
+          expect(errors.first[:source]).to eq('/veteranIdentification/mailingAddress/country')
+          expect(errors.first[:title]).to eq('Invalid country')
+          expect(errors.first[:detail]).to eq('Provided country is not valid: INVALID_COUNTRY')
+        end
+      end
+
+      context 'when BGS service is unavailable' do
+        let(:form_attributes) do
+          base_form_attributes.merge(
+            'veteranIdentification' => {
+              'mailingAddress' => {
+                'addressLine1' => '123 Main St',
+                'city' => 'London',
+                'country' => 'GBR',
+                'internationalPostalCode' => 'SW1A 1AA'
+              }
+            }
+          )
+        end
+
+        before do
+          allow_any_instance_of(described_class).to receive(:valid_countries).and_return(nil)
+        end
+
+        it 'returns BGS service error' do
```

Comment:
Looks like the confluence doc references incorrect service here: this should be a BRD service error, not BGS.